### PR TITLE
Add documentation to project

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**
dont merge yet

### CODE REVIEW

**Comment by gabrieltorresgamez** [comment]
Have a look and tell me if it looks ok:
https://github.com/Sensor-Based-Activity-Recognition/docs

For changes in this Repo you should look at this branch, not the main branch.
https://github.com/Sensor-Based-Activity-Recognition/pipelines/tree/add-documentation

Don't question the commit names

**Comment by gabrieltorresgamez** [comment]
Closes https://github.com/Sensor-Based-Activity-Recognition/docs/issues/7

File: README.md [hunk 1]
```diff
@@ -1,23 +1,76 @@
...
+Pipelines ist unser Repository für unsere Data-Processing-Pipelines und unseren trainierten Modelle. 
+Die Pipelines werden mit DVC verwaltet und können mit DVC reproduziert werden.
 
-# How to DVC
-## Install DVC
+## Kollaboration
+Um neue Funktionen zu implementieren, muss ein neuer Branch erstellt werden.
+Dieser soll danach von einem anderen Teammitglied reviewed werden.
+Nicht funktionierende Pipelines (Fehler im Workflow) sollen nicht gemerged werden.
+Daten müssen nicht umbedingt mit DVC gepusht werden, da der Workflow dies bereits übernimmt.
```
Comment (yvokeller) [inline README.md]: unbedingt

**Review by yvokeller — APPROVED** [review]
I like

**Pull request merged**